api/libproject/devstart.rb

```diff
@@ -1031,7 +1092,7 @@ def generate_cb_criteria_tables(cmd_name, *args)
 })
 
 def generate_private_cdr_counts(cmd_name, *args)
-  ensure_docker_sync()
+  ensure_docker cmd_name, args
```

Comment:
Do we need ensure_docker now that we will be running this on circle-ci outside of docker ?

Reply:
We need docker because this line runs the liquibase build: https://github.com/all-of-us/workbench/blob/5527c8980c986d5e93d342c4db88f41e6cb2dbba/api/db-cdr/generate-cdr/generate-private-cdr-counts.sh#L38-L38

Reply:
How does your build create the database and tables?

Reply:
I missed this. Running liquibase is part of import script in databrowser which is different step from building cdr indices.